template/libtofs.jinja

```diff
@@ -5,8 +5,8 @@
                        v1_path_prefix='') %}
   {#-
     Returns a valid value for the "source" parameter of a "file.managed"
-    state function. This makes easier the usage of the Template Override and
-    Files Switch (TOFS) pattern.
+    state function. This makes easier the usage of the "Template Override and
+    Files Switch" (TOFS) pattern.
```

Comment:
Is there a specific reason for this change?  Otherwise, this will lead to unnecessary false positives when comparing across formulas -- there are a lot of `libtofs.jinja` files out there now.

Reply:
Sorry then, I've reverted this and removed quotations.